### ISSUE TITLE
Someway to add in window shortcuts to the mono version?

### ISSUE THREAD

**Issue by Guille-814** [topeterk/HitCounterManager#34]
Hi,
I use linux and I would like to use shortcuts, I know that implementing global hotkeys on linux is hard, however I imaging that implementing shortcuts that works on the window is easier and that will make it easier to manually configure global hotkeys (for example with hyprland there is an option that passes the shortcut to the specified window). I wonder if there is a way to make this work by changing some configuration or something like that
Thanks

**Comment by topeterk** [comment]
Hi, I looked around a lot at the beginning of the project, how I can do that on Linux, but due to the variety of desktop solutions there are tons of different implementations with almost nothing in common. This results in endless different solutions to get it working "everywhere". When when looking on some "big players" like Gnome, Kde, Wayland or X, it is a pain getting global hotkeys to work. Looking a bit into some of the solutions (at least during the time I researched this stuff) I just end up in eternal mailing lists about bugs and issues. There are also lots of undocumented "solutions" (that may work or not, who knows), fishy workarounds or even required patches to the kernel. Most likely a normal user will not have a patched kernel or is not willing to rebuild his/her kernel again. So I eventually gave up on this.
Maybe this has changed in the meantime but it still feels like there is still no streamlined implementation available, so it requires special solutions for each desktop platform.
I guess, I will eventually loot at it again one day but I don't think in the near future.

**Comment by Guille-814** [comment]
Yeah, I know that's why I suggested the "in window" solution even tho is not a global hotkey, but I understand that is probably harder than it looks. Anyway thank you for making this project and for the early response, have a good day :D

**Comment by topeterk** [comment]
You thought about hotkeys while the HCM application has focus, right?
Such a thing could be possible with just using .Net, so yes, I made a note about it.
It will still "lack" the global hotkeys feature but, yes sure, then one could use the keyboard instead of the mouse to do actions within in GUI.
Thank you, much appreciated :)

**Comment by Guille-814** [comment]
Yeah that's what I mean, you're welcome and thanks to you for your time keep up the good work :D
